src/FluentCMS.Web.UI/Helper.cs

```diff
@@ -30,4 +38,58 @@ public static class Helper
 
         return default;
     }
+
+    public static async Task SignInAsync(this HttpContext httpContext, UserLoginRequest request)
+    {
+        var scopeFactory = httpContext.RequestServices.GetRequiredService<IServiceScopeFactory>();
+        using var scope = scopeFactory.CreateScope();
+        var serviceProvider = scope.ServiceProvider;
+        var accountClient = serviceProvider.GetRequiredService<AccountClient>();
+
+        //get access token
+        var loginResponseIApiResult = await accountClient.AuthenticateAsync(new UserLoginRequest()
+        {
+            Username = request.Username,
+            Password = request.Password
+        });
+
+        var claims = new List<Claim>();
+
+        //force set auth header
+        // todo: find a better solution for this
+        var httpClient = (HttpClient) accountClient.GetType().GetField("_httpClient", BindingFlags.NonPublic | BindingFlags.Instance)!.GetValue(accountClient)!;
```

Comment:
@pournasserian @babakhani @abdolian @TheHadiAhmadi any idea how to improve this?